lint-staged.config.js

```diff
@@ -1,14 +1,9 @@
 module.exports = {
-  '**/*.{ts,tsx,js,jsx}': (files) => {
+  '*.{ts,tsx,js,jsx}': (files) => {
     return [
-      `import-sort --write ${files.join(' ')}`,
-      `utsdx lint --max-warnings 0 ${files.join(
-        ' ',
-      )} --ignore-pattern '!.storybook'`,
+      `eslint --max-warnings 0 ${files.join(' ')}`,
       'lerna run type:check',
     ];
   },
```

Comment:
"*.{ts,tsx,js,jsx}":
        "yarn eslint --max-warnings 0" && lerna run type:check,

Reply:
that wouldnt work